roles/router/tasks/uninstall.yml

```diff
@@ -3,9 +3,9 @@
 - set_fact:
     systemd_network_dir: /etc/systemd/network
 
-- name: "remove a static route on srsran host for {{ core.upf.ip_prefix }} via {{ core.amf.ip }}"
+- name: "remove a static route on srsran host for {{ core.upf.access_subnet }} via {{ core.amf.ip }}"
   shell: |
-    ip route del {{ core.upf.ip_prefix }} via {{ core.amf.ip }}
+    ip route del {{ core.upf.access_subnet }} via {{ core.amf.ip }}
```

Comment:
Same as the comment in the `install` tasks, this is incorrect because it is not possible to add a route to an IP address that includes its mask